ai_user/prompts/image/base.py

```diff
@@ -38,8 +38,7 @@ async def _create_prompt(self, bot_prompt) -> Optional[list[dict[str, str]]]:
         return prompt
 
     async def _process_image(self, image: Image, bot_prompt: str) -> Optional[list[dict[str, str]]]:
-        raise NotImplementedError(
-            "_process_image() must be implemented in subclasses")
+        raise NotImplementedError("_process_image() must be implemented in subclasses")
```

Comment:
What autoformatter are you using?
I can switch to avoid this sorta changes

Reply:
Oh feel free to undo those. I tend to compact lines as long as they don't go past 120 characters wide

Reply:
I use PyCharm for the most part, but it doesn't reformat code for me unless I paste it in

Reply:
Should be fine, I'll try to change the autoformatter settings